clients/fossid-webapp/src/test/assets/report/mappings/xlsx_report.json

```diff
@@ -0,0 +1,30 @@
+{
+  "id" : "be42f1f5-7093-4370-89f0-5db14f8c0485",
+  "name" : "apiphp",
+  "request" : {
+    "url" : "/api.php",
+    "method" : "POST",
+    "bodyPatterns" : [ {
+      "equalToJson" : "{\"action\":\"generate_report\",\"group\":\"scans\",\"data\":{\"username\":\"\",\"key\":\"\",\"scan_code\":\"semver4j_semver4j__20220119_094101_delta\",\"report_type\":\"xlsx\",\"selection_type\":\"include_foss\"}}",
+      "ignoreArrayOrder" : true,
+      "ignoreExtraElements" : true
+    } ]
+  },
+  "response" : {
+    "status" : 200,
+    "bodyFileName" : "generate_report-1650976369669.xlsx",
+    "headers" : {
+      "Cache-Control" : "no-cache",
+      "Content-Disposition" : "attachment; filename=semver4j_semver4j__20220119_094101_delta-4d39359cd861c314020ead401d95ca20dc179495.xlsx",
```

Comment:
BTW, do you have any idea how the `filename` here relates to the `bodyFileName` in line 15?

Reply:
The `filename` is the special header returned by FossID.
`bodyFileName` is Wiremock specify trick to reference a file payload to the response your are stubbing. This is mainly used to avoid having binary in the JSON stub.

Reply:
Thanks for the explanation. So the `bodyFileName` part does not come from FossID itself, that's what I was after.